net-analyzer/openbsd-netcat/openbsd-netcat-1.190.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="The OpenBSD network swiss army knife"
+HOMEPAGE="https://cvsweb.openbsd.org/src/usr.bin/nc/
+	https://salsa.debian.org/debian/netcat-openbsd"
+SRC_URI="http://http.debian.net/debian/pool/main/n/netcat-openbsd/netcat-openbsd_${PV}.orig.tar.gz
+	http://http.debian.net/debian/pool/main/n/netcat-openbsd/netcat-openbsd_${PV}-2.debian.tar.xz"
+LICENSE="BSD"
+SLOT="0"
+IUSE="elibc_Darwin"
+
+KEYWORDS="~amd64 ~ppc64 ~x86 ~amd64-linux ~x64-macos"
+
+DEPEND="virtual/pkgconfig"
+RDEPEND="!elibc_Darwin? ( dev-libs/libbsd )
+	!net-analyzer/netcat
+	!net-analyzer/netcat6
+"
+
+S=${WORKDIR}/netcat-openbsd-${PV}
+
+src_prepare() {
+	for i_patch in $(<"${WORKDIR}/debian/patches/series"); do
+		eapply "${WORKDIR}/debian/patches/${i_patch}"
+	done
+	if [[ ${CHOST} == *-darwin* ]] ; then
+		# this undoes some of the Debian/Linux changes
+		eapply -p0 "${FILESDIR}"/${P}-darwin.patch
```

Comment:
rather make the patch to use `-p1`

Reply:
I preserved it for "minimum change". Now updated.